bus_uri: /dev/i2c-1
mpu_address: 0x68
pub_rate: 200
frame_id: imu
axes_offsets: [0, 0, 0, 0, 0, 0]
ki: 0.20000000000000001
kp: 0.10000000000000001
delta: 0.20000000000000001
Xgain: 0.99591491699218748
YtoX: -0.0031604003906250001
ZtoX: 0.081433105468750003
Xofs: -0.00078877766927083336
XtoY: -0.0031121826171875001
Ygain: 0.99959594726562506
ZtoY: 0.0064855957031250003
Yofs: -0.0081247965494791671
XtoZ: -0.098767089843750003
YtoZ: -0.032595825195312497
Zgain: 1.0119696044921875
Zofs: -0.00021769205729166668
gyro_x_offset: -2.460000010728836
gyro_y_offset: 8.7046183967590327
gyro_z_offset: 1.5921755743026733